src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -104,9 +104,9 @@ public MultiTable convert(Model model, String pathname) throws IOException {
 
       Map<String, Integer> nameToColumn = new HashMap<String, Integer>();
       i = 0;
-      for (String head : header) {
-        if (!head.equalsIgnoreCase(data.getTimeName())) {
-          newHead[i++] = head.trim();
+      for (int p = 1; p < header.length; p++) {
```

Comment:
Please add a comment why this loop is starting at 1. What are you ignoring?

Reply:
@matthiaskoenig, the time column has to be ignored for the newHead array. So, its started at index 1.

Reply:
This was the main reason why the travis ignored the SBMLTestSuiteTest file.

Reply:
Please just add a comment. Here, it can be as simple as adding `// we are starting at 1 because...` in the line over the loop.

Reply:
@draeger, this is commented above at line 100 in this file: `// exclude time column`.